devito/operator.py

```diff
@@ -280,6 +283,7 @@ def cfunction(self):
         if self._cfunction is None:
             self._cfunction = getattr(self._lib, self.name)
             argtypes = [c_int if isinstance(v, ScalarArgument) else
+                        c_float if isinstance(v, ConstantArgument) else
```

Comment:
Just a reminder that here we're hardcoding assumptions about the data types of ScalarArgument and ConstantArgument